### PR TITLE
Chore: Add more dummy SNS proposals

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Testnet users should create more types of dummy SNS proposals for better testing.

# Changes

* Add more dummy proposals in sns-dummy.api.
* Use allSettled and log individual fails for better debugging in testnet.

# Tests

* Not applicable. Functionality only for testnet.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@dskloetd @mstrasinskis @peterpeterparker please review

File: frontend/src/lib/api/dev.api.ts [hunk 1]
```diff
@@ -148,17 +148,29 @@ export const makeSnsDummyProposals = async ({
...
+            proposal.title
+          );
+          console.error(error);
+          throw error;
+        })
     )
   );
 
+  if (allCalls.some((call) => call.status === "rejected")) {
+    throw new Error();
```
Comment (dskloetd) [inline frontend/src/lib/api/dev.api.ts:172]: Would it be useful to include the errors from the failed calls in the error? Or are they visible in some other way?

**Review by dskloetd — APPROVED** [review]
I'm OK with not testing this, but it's not true that testing is not applicable.
If the functionality is useful, it would be nice for it to not break.
And things that aren't tested tend to break eventually.

File: frontend/src/lib/api/dev.api.ts (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/lib/api/dev.api.ts:172]: I console log them before.

And the service catching the error already shows an error saying that "something went wrong, please check the console for details"

**Pull request merged**